packages/hierarchies-react/src/presentation-hierarchies-react/UseTree.ts

```diff
@@ -44,25 +61,34 @@ interface UseTreeResult {
    * or tree is reloading.
    */
   isLoading: boolean;
+  /**
+   * Provider used to load tree nodes.
+   */
+  hierarchyProvider?: HierarchyProvider;
```

Comment:
should this be marked `@internal`?

Reply:
Removed `hierarchyProvider` it was returned in order to get instance keys for filtering dialog. Added getter on `HierarchyLevelConfiguration` where all other stuff related to filtering dialog was.